packages/protocol/contracts/governance/Validators.sol

```diff
@@ -1131,4 +1147,57 @@ contract Validators is
       }
     }
   }
+
+  /**
+   * @notice Sets the slashingMultiplierRestPeriod property if called by owner.
+   * @param value New reset period for slashing multiplier.
+   */
+  function setSlashingMultiplierResetPeriod(uint256 value) public nonReentrant onlyOwner {
+    slashingPenaltyResetPeriod = value;
+  }
+
+  /**
+   * @notice Resets a group's slashing multiplier if it has been >= the reset period since 
+   *         the last time the group was slashed.
+   */
+  function resetSlashingMultiplier() external nonReentrant {
+    address account = getAccounts().validatorSignerToAccount(msg.sender);
+    require(isValidatorGroup(account));
+    ValidatorGroup storage group = groups[account];
+    require(
+      now >= group.slashInfo.lastSlashed.add(slashingPenaltyResetPeriod),
+      "`resetSlashingMultiplier` called before resetPeriod expired"
+    );
+    group.slashInfo.penalty = FixidityLib.fixed1();
+  }
+
+  bytes32[] canHalveSlashingMultiplier = [
+    DOWNTIME_SLASHER_REGISTRY_ID,
+    DOUBLE_SIGNING_SLASHER_REGISTRY_ID
+  ];
+
+  /**
+   * @notice Halves the group's slashing multiplier.
+   * @param account The group being slashed.
+   */
+  function halveSlashingMultiplier(address account)
```

Comment:
Been thinking about this more, and wondering if it makes sense for different slashers to set their own multiplier between 0 and 1, let's discuss more offline